manager2/src/app/user/user.service.ts

```diff
@@ -256,7 +258,13 @@ export class UserService {
     };
     return this.http.get(
       environment.apiUrl + '/user',
-      httpOptions)    
+      httpOptions).pipe(map((response: any) => {
+          return response.map(item => {
+            return item;
+          }).sort(function (a,b) {
```

Comment:
why a response.map(item => return item) ?? If I am correct, this will return a copy of response.

Reply:
Because it was done like this in groups services : https://github.com/genouest/genouestaccountmanager/blob/cf900751733845be443455ee457d9d366767bc38/manager2/src/app/admin/groups/groups.service.ts#L27

You are right, it is useless, and it look like it is cleaner to add some [] to first map and apply the sort on response: https://github.com/genouest/genouestaccountmanager/pull/91/commits/6a911bdd6e8bb5dd00bf134a2f3c46cc8d23276c